.gitignore

```diff
@@ -11,3 +11,4 @@ site/script/target
 site/target
 planck-cljs/tools.reader
 planck-cljs/clojurescript
+.idea
```

Comment:
This is cool. It could probably be a wholly-separate commit as it is unrelated to the quiet mode.

By the way, I have this in my `~/.gitignore` so that it works for all projects I work on or contribute to, without having to bother adding it to individual projects:

```
*.iml
.idea/
```